VideoLocker/res/values/strings.xml

```diff
@@ -508,7 +508,7 @@
     <!-- Button title for action that allows users to share this app -->
     <string name="button_share_app">Share app to friends</string>
     <!-- Alert dialog title for unknown type of error -->
-    <string name="error_unknown">Unknown Error</string>
+    <string name="error_unknown_title">Unknown Error</string>
```

Comment:
What's the point in having two different messages for unknown errors? Also why "title"? Also note that `"There was an unknown error"` is hardcoded quite a few times in `FacebookProvider`.